common/client-core/src/client/base_client/mod.rs

```diff
@@ -622,6 +630,21 @@ where
         mix_tx
     }
 
+    fn start_mix_traffic_controller_with_forget_me(
```

Comment:
tiny nitpick: why define separate function for essentially the same thing? just have a `MixTrafficController::new()` that  takes `forget_me` flag which would be `false` by default